### PR TITLE
Branch1

### PR DESCRIPTION
**Pull request by srinivaskarne**
please refer @casey-collab

### CODE REVIEW

**Comment by pmallory** [comment]
@casey-collab can you take a look?

File: language.md [hunk 1]
```diff
@@ -135,3 +135,5 @@ As jy Afrikaans kan praat, gaan [hier](afrikaans/kos.md)
 Kama unazungumza kiswahili [ingia hapa](swahili/marshmallow.md)
 
 If you speak Gujarati, go [here](Gujarati/gujarati.md).
+If you learn Telugu, click [here](telugu/hyderabad-telugu/telugu.md).
```
Comment (casey-collab) [inline language.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**